packages/next/src/client/app-dir/link.tsx

```diff
@@ -299,6 +299,36 @@ function formatStringOrUrl(urlObjOrString: UrlObject | string): string {
   return formatUrl(urlObjOrString)
 }
 
+/**
+ * Returns the ref of a React element handling differences between React 19 and older versions.
```

Comment:
Isn't this link solely used in App Router where we don't need to handle React 18?